src/ctia/stores/es/store.clj

```diff
@@ -294,4 +294,4 @@
     (inv/handle-list state filtermap ident params))
   IQueryStringSearchableStore
   (query-string-search [_ query filtermap ident params]
-    (tool/handle-query-string-search state query filtermap ident params)))
+    (inv/handle-query-string-search state query filtermap ident params)))
```

Comment:
@gbuisson Gets credit for finding this stupid bug.  I did a bad cut-and-paste.  :(